tests/lib/rules/component-tags-order.js

```diff
@@ -268,7 +345,23 @@ tester.run('component-tags-order', rule, {
           message: 'The <script> should be above the <style> on line 2.',
           line: 3
         }
-      ]
+      ],
+      output: '\n        <script></script>\n        <style></style>\n      '
     }
   ]
 })
+
+describe('suppress reporting with eslint-disable-next-line', () => {
+  it('disable all rules if <!-- eslint-disable-line -->', async () => {
```

Comment:
The test name should probably be adjusted.

Reply:
Nice catch! It was copied from another test and I forgot to modify the test description.

Reply:
Updated.